### PR TITLE
[Outreachy Task Submission]: Add Delete Icon [Survey]

### PR DESCRIPTION
**Pull request by Cecil-o**
Added Delete Icon to Survey, this solve issue  #4894

-These allow user to delete a survey when a single action is require for ease and fast navigation.

-I used font awesome icon in carrying out this task.
survey font
survey

### CODE REVIEW

File: apps/web-mzima-client/src/env.json
Comment (tuxpiper) [inline apps/web-mzima-client/src/env.json:1]: Please remove this change, it's not useful to others

**Review by tuxpiper — CHANGES_REQUESTED** [review]
Thank you for the contribution!

Could you please add some more details to the issue that you created (ushahidi/platform#4894)? It's difficult to follow and make sure I understand what you are intending to fix here.

**Comment by tuxpiper** [comment]
> Thank you for the contribution!
> 
> Could you please add some more details to the issue that you created ([ushahidi/platform#4894](https://github.com/ushahidi/platform/issues/4894))? It's difficult to follow and make sure I understand what you are intending to fix here.

A screenshot would be helpful

**Comment by Cecil-o** [comment]
> Thank you for the contribution!
> 
> Could you please add some more details to the issue that you created ([ushahidi/platform#4894](https://github.com/ushahidi/platform/issues/4894))? It's difficult to follow and make sure I understand what you are intending to fix here.

Hello, my apology for the delay, had some technical issue. The changes made should be clear now. Thanks

**Pull request closed without merging**